Thomascountz/relay
# Text Editor

Closes #6 

## Description
Render a text area so that a user can input text. Provide context menu for copying, pasting, cutting, selecting, undoing, and redoing text.

## How to Test
Unfortunately, Spectron, our E2E testing framework, doesn't have an API for testing context menus, (see here: https://github.com/electron/spectron/issues/21), so that needs to be tested manually.

Otherwise, running jest test (`npm test`) and spectron tests (`npm run-script e2e_tests`) will verify that the app renders a text box without crashing.
